alchemist-mix.el

```diff
@@ -37,6 +37,11 @@
   :type 'string
   :group 'alchemist-mix)
 
+(defcustom alchemist-mix-test-default-options '("--exclude pending:true")
+  "Default options for alchemist test command."
+  :type 'list
```

Comment:
I tried running this and was getting issues when trying to change the variable with the list type -- I think you actually want a composite type that looks like:
```
:type '(repeat string)
```
See custom's [composite types](https://www.gnu.org/software/emacs/manual/html_node/elisp/Composite-Types.html#Composite-Types).